src/demo/containers/Demo.tsx

```diff
@@ -1,144 +1,151 @@
+class Demo extends React.Component<{}, DemoState> {
   /**
    * Maintain internal state of user session
    */
   constructor(props: any) {
-    super(props);
+    super(props)
     this.state = {
-      token: "",
-      version: 0,
-      ownerId: "",
       isLoading: true,
+      ownerId: '',
       showMarkdown: true,
-    };
-    this.makeSampleQueryCall = this.makeSampleQueryCall.bind(this);
-    this.getVersion = this.getVersion.bind(this);
-    this.handleChange = this.handleChange.bind(this);
-    this.removeHandler = this.removeHandler.bind(this);
+      token: '',
+      version: 0
+    }
+    this.makeSampleQueryCall = this.makeSampleQueryCall.bind(this)
+    this.getVersion = this.getVersion.bind(this)
+    this.handleChange = this.handleChange.bind(this)
+    this.removeHandler = this.removeHandler.bind(this)
   }
   /**
    * Get the current version of Synapse
    */
-  getVersion(): void {
+  public getVersion(): void {
     // IMPORTANT: Your component should have a property (with default) to change the endpoint.  This is necessary for Synapse.org integration.
     // Pass your endpoint through to the rpc call:
     // SynapseClient.getVersion('https://repo-staging.prod.sagebase.org')
     SynapseClient.getVersion()
-      .then((data: SynapseVersion) => this.setState({version: data.version}))
-      .catch(function(error: any) {
+      .then((data: SynapseVersion) => this.setState({ version: data.version }))
+      .catch((error: any) => {
         // Handle HTTPError.  Has statusCode and message.
-        console.error("Get version failed", error);
-      });
+        console.error('Get version failed', error)
+      })
   }
   /**
    * Make a query on synapse
    */
-  makeSampleQueryCall(): void {
+  public makeSampleQueryCall(): void {
     // Example table (view) query.
     // See https://docs.synapse.org/rest/POST/entity/id/table/query/async/start.html
-    let QUERY = {
-      entityId: "syn12335586",
+    const QUERY = {
+      entityId: 'syn12335586',
+      // like it or not synapse has a bitwise protocol for parmask
+      partMask:
+      // tslint:disable-next-line
+        SynapseConstants.BUNDLE_MASK_QUERY_RESULTS |
+        SynapseConstants.BUNDLE_MASK_QUERY_COLUMN_MODELS |
+        SynapseConstants.BUNDLE_MASK_QUERY_SELECT_COLUMNS |
+        SynapseConstants.BUNDLE_MASK_QUERY_FACETS,
       query: {
-        sql: "SELECT * FROM syn12335586",
         includeEntityEtag: true,
         isConsistent: false,
+        limit: 100,
         offset: 0,
-        limit: 100
+        sql: 'SELECT * FROM syn12335586'
       },
-      partMask:
-        SynapseConstants.BUNDLE_MASK_QUERY_RESULTS |
-        SynapseConstants.BUNDLE_MASK_QUERY_COLUMN_MODELS |
-        SynapseConstants.BUNDLE_MASK_QUERY_SELECT_COLUMNS |
-        SynapseConstants.BUNDLE_MASK_QUERY_FACETS
-    };
+    }
     SynapseClient.getQueryTableResults(QUERY)
       .then((data: any) => console.log(data))
-      .catch(function(error: any) {
```

Comment:
Please create a Confluence wiki page to document local setup (VS Code formatter, for example)